src/integration_tests/test_validation.py

```diff
@@ -21,8 +21,6 @@ def test_validation(self, project_name):
 class TestValidationElife():
     @classmethod
     def setup_class(cls):
-        # HERE BE DRAGONS
-        # resets the testing config.SETTINGS_FILE we set in the base.BaseCase class
```

Comment:
eh, isn't there something still dodgy here? or was that fixed?

Reply:
it is now changing `config.PROJECT_FILES`, but it seems to have become an established pattern, so lest of introducing dependency injection it's here to stay